### PR TITLE
nmk16.cpp: Fix 'redhawks' bad graphics and add placeholders for undumped PROMs and PLDs

### PR DESCRIPTION
**Pull request by clawgrip**

### CODE REVIEW

**Comment by clawgrip** [comment]
redhawk_mame
redhawk_pcb

Also, see https://youtu.be/wRI-GW6LD-o for reference

File: src/mame/drivers/nmk16.cpp [hunk 1]
```diff
@@ -8658,7 +8666,7 @@ GAME( 1995, dolmen,     0,        twinactn,     dolmen,       nmk16_state, init_
 GAME( 1998, stagger1,   0,        stagger1,     stagger1,     afega_state, empty_init,           ROT270,             "Afega",                             "Stagger I (Japan)", 0 )
 GAME( 1997, redhawk,    stagger1, stagger1,     stagger1,     afega_state, init_redhawk,         ROT270,             "Afega (New Vision Ent. license)",   "Red Hawk (USA, Canada & South America)", 0 )
 GAME( 1997, redhawki,   stagger1, redhawki,     stagger1,     afega_state, init_redhawki,        ROT0,               "Afega (Hae Dong Corp license)",     "Red Hawk (horizontal, Italy)", 0 ) // bootleg? strange scroll regs
-GAME( 1997, redhawks,   stagger1, stagger1,     stagger1,     afega_state, empty_init,           ROT0,               "Afega (Hae Dong Corp license)",     "Red Hawk (horizontal, Spain)", 0 )
+GAME( 1997, redhawks,   stagger1, stagger1,     stagger1,     afega_state, empty_init,           ROT0,               "Afega (Hae Dong Corp license)",     "Red Hawk (horizontal, Spain)", GAME_IMPERFECT_GRAPHICS )
```
Comment (cuavas) [inline src/mame/drivers/nmk16.cpp:8669]: This won’t build, as shown by the CI failures.

**Review by cuavas — CHANGES_REQUESTED** [review]

File: src/mame/drivers/nmk16.cpp (hunk 1 above)
Comment (cam900) [inline src/mame/drivers/nmk16.cpp:8669]: Please use MACHINE_IMPERFECT_GRAPHICS instead in here.

**Comment by Osso13** [comment]
Try changing the used machine_config() from stagger1 to redhawki. It seems to fix the problems. Also Hae Dong is the producer of both redhawki and redhawks, so it would IMHO make sense they share the same code base.

**Comment by clawgrip** [comment]
It works fine changing from stagger1 to redhawki Don't know what happened the first time I tested it that I had to use stagger1, for sure my bad.

File: src/mame/drivers/nmk16.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/drivers/nmk16.cpp]: It's fixed now, thanks

Comment (clawgrip) [reply src/mame/drivers/nmk16.cpp]: Thx!

**Pull request merged**